src/src/FHSS.cpp

```diff
@@ -80,32 +93,41 @@ void ICACHE_RAM_ATTR FHSSrandomiseFHSSsequence()
     // sync channels
     const int SYNC_INTERVAL = 20;
 
-    int nLeft = NR_FHSS_ENTRIES-1; // how many channels are left to be allocated. Does not include the sync channel
-    unsigned int prev = 0; // needed to prevent repeats of the same index
+    int nLeft = NR_FHSS_ENTRIES - 1; // how many channels are left to be allocated. Does not include the sync channel
+    unsigned int prev = 0;           // needed to prevent repeats of the same index
 
     // for each slot in the sequence table
-    for(int i=0; i<NR_SEQUENCE_ENTRIES; i++) {
-        if (i % SYNC_INTERVAL == 0) {
+    for (int i = 0; i < NR_SEQUENCE_ENTRIES; i++)
+    {
+        if (i % SYNC_INTERVAL == 0)
+        {
             // assign sync channel 0
             FHSSsequence[i] = 0;
             prev = 0;
-        } else {
+        }
+        else
+        {
             // pick one of the available channels. May need to loop to avoid repeats
             unsigned int index;
-            do {
+            do
```

Comment:
wow... dont think Ive ever seen anyone use a do/while loop before...

Reply:
That is James' code :)